rviz_rendering/CMakeLists.txt

```
@@ -165,10 +165,12 @@ if(BUILD_TESTING)
   find_package(ament_cmake_gmock REQUIRED)
   find_package(ament_cmake_gtest REQUIRED)
 
-  add_library(rviz_rendering_test_utils
+  add_library(rviz_rendering_test_utils SHARED
```

```suggestion
  add_library(rviz_rendering_test_utils
```

(it's not necessary anymore since it is indeed the default for `ament_cmake_ros`)